v3/src/models/formula/functions/math.ts

```diff
@@ -88,7 +89,9 @@ export const fnRegistry = {
 
   ...bivariateStatsFunctions,
 
-  ...semiAggregateFunctions
+  ...semiAggregateFunctions,
+
+  ...colorFunctions
```

Comment:
There's a loose simple to complex ordering going on here, so I'd put this up between the logic functions and the date functions with the other type-specific functions.